### PR TITLE
Jenkinsfile changes for #7333.

### PR DESCRIPTION
**Pull request by areusch**
* Make build/standalone_crt available to CI unit tests.
 * We could do this with stash but it's complex because we would need
   to use wildcards. Instead, rebuild standalone_crt source tree before testing.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -248,6 +248,8 @@ stage('Unit Test') {
         unpack_lib('gpu', tvm_multilib)
         timeout(time: max_time, unit: 'MINUTES') {
           sh "${docker_run} ${ci_gpu} ./tests/scripts/task_ci_python_setup.sh"
+          sh "${docker_run} ${ci_gpu} ./tests/scripts/task_config_build_gpu.sh"
+          sh "${docker_run} ${ci_gpu} ./tests/scripts/task_build.sh build standalone_crt"
```
Comment (tqchen) [inline Jenkinsfile:252]: Would this trigger rebuild of libtvm? or simply just `standalone_crt`?

Comment (areusch) [reply Jenkinsfile]: ok resolved per our discussion. we will not explicitly introduce a separate build step here; instead we will add this build to `tests/scripts/task_ci_setup.sh`, which was previously `tests/scripts/task_ci_python_setup.sh`

**Comment by areusch** [comment]
@tqchen good to merge now?

**Comment by tqchen** [comment]
@areusch please resolve merge conflict

**Comment by areusch** [comment]
@tqchen done

**Review by tqchen — APPROVED** [review]

**Pull request merged**